pt-BR/1-5.md

```diff
@@ -0,0 +1,86 @@
+
+Só resta uma pequena coisa para fazer antes que você possa dar uma olhada no resultado: temos que carregar a imagem dentro do ImageView do `DetailViewController`.
+
+Este novo código irá utilizar um novo tipo de objeto chamado `UIImage`. Este tipo não tem um "View" no seu nome como no caso do `UIImageVIew`, portanto não é algo que possa ser visto pelos usuários. Ao invés disto, `UIImage` é um tipo de dado que irá armazenar as informações que fazem um imagem, como PNG ou JPG.
+
+Uma das alternativas para criar uma `UIImage` é passar um parâmetro com o nome de `named` que permite você definir qual imagem deve ter seus dados lidos. Neste caso o tipo `UIImage` vai buscar por este nome dentro do pacote da sua app e carregá-la. Usando o nome contido na propriedade `selectedImage` que foi preenchida pelo `ViewController`, a imagem que será carregada será a imagem escolhida pelo usuário.
+
+Entretanto, nós não conseguiremos usar `selectedImage` diretamente. Se você se lembra, nós definimos a propriedade da seguinte maneira:
+
+    var selectedImage: String?
+
+Este `?` significa que ele pode ter um valor, ou não. E a linguagem Swift não permite que você use propriedades caracterizadas por este "talvez" sem primeiro checar sua validade. Esta é uma nova oportunidade para usar o `if let`: nós podemos checar se `selectedImage` possui um valor e, em caso positivo, usar este valor. Do contrário, não fazer nada.
+
+Adicione o código seguinte ao método `viewDidLoad()` que pertence ao `DetailViewController`, *após* a chamada `super.viewDidLoad()`:
+
+    if let imageToLoad = selectedImage {
+        imageView.image  = UIImage(named: imageToLoad)
+    }
+
+A primeira linha checa e desembrulha o valor dentro da propriedae `selectedImage`. Se por alguma razão `selectedImage` é nulo (o que em teoria nunca vai acontecer), então a instrução `imageView.image` nunca será executada. Se por outro lado ela possuir um valor este será incluído na variável `imageToLoad` e passado como parametro para `UIImage` e então carregado.
+
+Ok, é isto: pressione play ou Cmd+R para executar a aplicação e testar o novo código. Você deve ser capaz de selecionar qualquer uma das figuras para que elas possam entrar deslizando para serem mostradas em tela cheia.
+
+Perceba que apareceu um botão Voltar na barra de navegação que permite ao usuário retornar ao `ViewController`. Se você clicar e arrastar com delicadeza, perceberá que consegue fazer a mesma operação usando gestos de deslizar - pressione o extremo esquerdo da tela e deslize para a direita, como você faria usando o telefone.
```

Comment:
Actually the user will see "Back" not "Voltar" on the navigation bar. You can translate if you want. But I think you should mention what the user will see.